source/includes/api/_dataset.md

```diff
@@ -74,81 +76,140 @@ curl -X GET https://api.resourcewatch.org/v1/dataset
-app           | Applications to which this dataset is being used                              | Available applications like: `["data4sdgs","gfw","rw","aqueduct","prep","forest-atlas","gfw-climate","aqueduct-water-risk","test","gfw-pro","globalforestwatch", "ghg-gdp"]`
-connectorType |                                                                              | `rest` or `document`
-provider      | Dataset provider this include inner connectors and 3rd party ones            | [A valid dataset provider](##supported-dataset-sources)
-userId        | the user who registered the dataset                                          | valid id
-user.role     | The role of the user who registered the dataset. If the requesting user does not have the ADMIN role, this filter is ignored. | `ADMIN`, `MANAGER` or `USER`
-status        | the internal dataset status at connection time                               | `pending`, `saved` or `failed`
-published     |                                                                              | `true`or `false`
-env           | If the dataset is in the staging, preproduction environment or in production one          | `staging`, `production` or `preproduction`. Defaults to `production`
-overwritted   | If the data can be overwritten (only for being able to make dataset updates) | `true`or `false`
-verify        | If this dataset contains data that is verified using blockchain              | `true`or `false`
-protected     | If it's a protected layer                                                    | `true`or `false`
-geoInfo       | If it contains interceptable geographical info                               | `true`or `false`
-subscribable  | If the dataset is subscribable (i.e. contains a non-empty object in the subscribable field) | `true` or `false`
+### Filters
 
 > Filtering datasets
 
 ```shell
 curl -X GET https://api.resourcewatch.org/v1/dataset?name=birds&provider=cartodb
 ```
 
-> For inclusive filtering with array props use '@'
+> Matching vocabulary tags
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/dataset?app=gfw@rw@prep
+curl -X GET https://api.resourcewatch.org/v1/dataset?vocabulary[legacy]=umd
 ```
 
-### Sorting
+The dataset list provides a wide range of parameters that you can use to tailor your dataset listing. Most of these parameters reflect fields you'll find in a dataset itself, while others are convenience filters for things like user role or favourites. These parameters can be combined into a complex `and` logic query.
 
-#### Basics of sorting
+Here's the comprehensive list of filters supported by datasets:
+ 
 
-The API currently supports sorting by means of the `sort` parameter. Sorting can be done using any field from the dataset, as well as `user.name` and `user.role` (sorting by user data is restricted to ADMIN users).
+Filter         | Description                                                                  | Type        | Expected values
```

Comment:
Missing some descriptions here. These need to be filled in. @ikas let me know if you need more time to track down some of these and I can make a separate task for it to return to later.

Reply:
Actually I was the one doing this. I already made some changes to improve this, but I think the main one here is the link to "dataset reference" at the top. Ultimately, I don't think it's useful to explain how a filter by fieldX works, I think we should explain what fieldX is, and the user can infer from there.